app/assets/stylesheets/govuk_publishing_components/components/helpers/_brand-colours.scss

```diff
@@ -8,6 +8,10 @@
     .brand--#{nth($organisation, 1)} {
       .brand__color {
         color: nth($organisation, 3);
+
+        &:hover {
+          color: lighten( nth($organisation, 3), 10% );
```

Comment:
For some colours, this will mean they no longer pass colour contrast checks.

**Top link hovered; bottom link normal:**
<img width="631" alt="screen shot 2018-06-12 at 13 16 43" src="https://user-images.githubusercontent.com/29889908/41289811-e034299a-6e42-11e8-9ea1-8f143bc36c6c.png">

<img width="717" alt="screen shot 2018-06-12 at 13 17 37" src="https://user-images.githubusercontent.com/29889908/41289855-007a0fc6-6e43-11e8-8489-a83214af28a9.png">

Reply:
The default link hover colour on GOV.UK fails that test too.

Reply:
Actually we can fix this easily if I go with 'darken' instead of 'lighten'. I'll do that.